cves/CVE-2005-3357.yml

```diff
@@ -40,8 +45,8 @@ bounty:
   amt:
   announced:
   url:
-reviews: []
-bugs: []
+reviews: [354394]
+bugs: [37791]
 repo:
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
```

Comment:
Even if the commits were provided for you, you should still look at the commits and make some kind of a note.